src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,37 @@
+package nextstep.qna.domain;
+
+import nextstep.qna.CannotDeleteException;
+import nextstep.users.domain.NsUser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Answers {
+    private final List<Answer> answers;
+
+    public Answers(List<Answer> answers) {
+        this.answers = new ArrayList<>(answers);
+    }
+
+    public void add(Answer answer) {
+        answers.add(answer);
+    }
+
+    public List<Answer> getAnswers() {
+        return answers;
+    }
+
+    public List<DeleteHistory> deleteAll(NsUser loginUser) {
+        for (Answer answer : answers) {
+            if (!answer.isOwner(loginUser)) {
+                throw new CannotDeleteException("다른 사람이 쓴 답변이 있어 삭제할 수 없습니다.");
+            }
+        }
+
+        List<DeleteHistory> deleteHistories = new ArrayList<>();
+        for (Answer answer : answers) {
+            deleteHistories.add(answer.delete());
+        }
+        return deleteHistories;
```

Comment:
```suggestion
        return answers.stream()
                .map(Answer::delete)
                .collect(Collectors.toList());
```
stream으로 표현해볼 수 있겠네요 😄